composer.json

```diff
@@ -89,7 +89,16 @@
     "extra": {
         "installer-name": "Framework",
         "neos": {
-            "warning": "AUTOGENERATED FILE, ONLY MODIFY THE .composer.json IN THIS DIRECTORY AND RUN THE MERGE SCRIPT."
+            "warning": "AUTOGENERATED FILE, ONLY MODIFY THE .composer.json IN THIS DIRECTORY AND RUN THE MERGE SCRIPT.",
+            "package-key": [
```

Comment:
I'm not so sure about this passage. Please tell me if I am supposed to cut it out.